prompt-service/entrypoint.sh

```diff
@@ -8,5 +8,4 @@
     --log-level debug \
     --timeout 900 \
     --access-logfile - \
-    --reload \
```

Comment:
@chandrasekharan-zipstack QQ : Does this param cause any issue?

Reply:
Its not required while running the docker container and it might help save some resources since we'll not have a thread / logic monitor changes maybe